modules/preview-react/pill/lib/Pill.tsx

```diff
@@ -65,50 +65,50 @@ const StyledBasePill = styled(BaseButton.as('button'))<StyledType & PillProps>(
-    [buttonVars.default.border]: colors.licorice200,
-    [buttonVars.default.label]: colors.blackPepper400,
+    [buttonColorPropVars.default.background]: colors.soap300,
+    [buttonColorPropVars.default.border]: colors.licorice200,
+    [buttonColorPropVars.default.label]: colors.blackPepper400,
     [systemIconStencil.vars.color]: colors.licorice200,
     button: {
       [systemIconStencil.vars.color]: colors.licorice200,
     },
     '&:focus-visible, &.focus': {
-      [buttonVars.focus.background]: colors.soap300,
-      [buttonVars.focus.border]: colors.blueberry400,
-      [buttonVars.focus.label]: colors.blackPepper400,
+      [buttonColorPropVars.focus.background]: colors.soap300,
+      [buttonColorPropVars.focus.border]: colors.blueberry400,
+      [buttonColorPropVars.focus.label]: colors.blackPepper400,
       [systemIconStencil.vars.color]: colors.licorice500,
       button: {
         [systemIconStencil.vars.color]: colors.licorice500,
       },
       'span[data-count="ck-pill-count"]': {
-        borderTop: `1px solid ${colors.blueberry400}`,
-        borderBottom: `1px solid ${colors.blueberry400}`,
-        borderRight: `1px solid ${colors.blueberry400}`,
+        borderTop: `${px2rem(1)} solid ${colors.blueberry400}`,
+        borderBottom: `${px2rem(1)} solid ${colors.blueberry400}`,
+        borderRight: `${px2rem(1)} solid ${colors.blueberry400}`,
       },
     },
     '&:hover, &.hover': {
-      [buttonVars.hover.background]: colors.soap400,
-      [buttonVars.hover.border]: colors.licorice400,
-      [buttonVars.hover.label]: colors.blackPepper400,
+      [buttonColorPropVars.hover.background]: colors.soap400,
+      [buttonColorPropVars.hover.border]: colors.licorice400,
+      [buttonColorPropVars.hover.label]: colors.blackPepper400,
       [systemIconStencil.vars.color]: colors.licorice500,
       button: {
         [systemIconStencil.vars.color]: colors.licorice500,
       },
     },
     '&:active, &.active': {
-      [buttonVars.active.background]: colors.soap500,
-      [buttonVars.active.border]: colors.licorice500,
-      [buttonVars.active.label]: colors.blackPepper400,
+      [buttonColorPropVars.active.background]: colors.soap500,
+      [buttonColorPropVars.active.border]: colors.licorice500,
+      [buttonColorPropVars.active.label]: colors.blackPepper400,
       [systemIconStencil.vars.color]: colors.licorice500,
       button: {
         [systemIconStencil.vars.color]: colors.licorice500,
       },
     },
     '&:disabled, &.disabled': {
-      [buttonVars.disabled.background]: colors.soap100,
-      [buttonVars.disabled.border]: colors.licorice100,
-      [buttonVars.disabled.label]: colors.licorice100,
-      [buttonVars.disabled.opacity]: '1',
+      [buttonColorPropVars.disabled.background]: colors.soap100,
+      [buttonColorPropVars.disabled.border]: colors.licorice100,
+      [buttonColorPropVars.disabled.label]: colors.licorice100,
+      [buttonColorPropVars.disabled.opacity]: '1',
```

Comment:
should this use `system.opacity.full` ? cc @RayRedGoose

Reply:
I wasn't sure if I was doing `Pill` in this refactor. I didn't want to refactor all of `Pill` because it belongs in it's own PR.

The updates I had to make were just because it uses the vars from `BaseButton`